dev/DynamicDependency/MsixDynamicDependency.h

```diff
@@ -0,0 +1,190 @@
+///       is removed Deployment won't remove Fwk-v2 because it's the only package
+///       satisfying the PackageDependency. Thus Fwk-v1 and Fwk-v2 (and any other
+///       package matching the PackageDependency) are 'loosely pinned'. Deployment
+///       guarantees it won't remove a package if it would make a PackageDependency
+///       unsatisfied.
+///
+/// A PackageDependency specifies criteria (package family, minimum version, etc)
+/// and not a specific package. Deployment reserves the right to use a different
+/// package (e.g. higher version) to satisfy the PackageDependency if/when
+/// one becomes available.
+///
+/// @param user the user scope of the package dependency. If NULL the caller's
+///        user context is used. MUST be NULL if MddCreatePackageDependencyOptions::ScopeIsSystem
+///        is specified
+/// @param lifetimeArtifact MUST be NULL if lifetimeKind=MddPackageDependencyLifetimeKind::Process
+/// @param packageDependencyId allocated via HeapAlloc; use HeapFree to deallocate
+///
+/// @note MddTryCreatePackageDependency() fails if the PackageDependency cannot be resolved to a specific
+///       package. This package resolution check is skipped if
+///       MddCreatePackageDependencyOptions::DoNotVerifyDependencyResolution is specified. This is useful
+///       for installers running as user contexts other than the target user (e.g. installers
+///       running as LocalSystem).
+STDAPI MddTryCreatePackageDependency(
+    PSID user,
+    _In_ PCWSTR packageFamilyName,
+    PACKAGE_VERSION minVersion,
+    MddPackageDependencyProcessorArchitectures packageDependencyProcessorArchitectures,
+    MddPackageDependencyLifetimeKind lifetimeKind,
+    PCWSTR lifetimeArtifact,
+    MddCreatePackageDependencyOptions options,
+    _Outptr_result_maybenull_ PWSTR* packageDependencyId) noexcept;
+
+/// Undefine a package dependency. Removing a pin on a PackageDependency is typically done at uninstall-time.
+/// This implicitly occurs if the package dependency's 'lifetime artifact' (specified via MddTryCreatePackageDependency)
+/// is deleted. Packages that are not referenced by other packages and have no pins are elegible to be removed.
+///
+/// @warn MddDeletePackageDependency() requires the caller have administrative privileges
+///       if the package dependency was pinned with MddCreatePackageDependencyOptions::ScopeIsSystem.
+STDAPI_(void) MddDeletePackageDependency(
+    _In_ PCWSTR packageDependencyId) noexcept;
+
+/// Resolve a previously-pinned PackageDependency to a specific package and
+/// add it to the invoking process' package graph. Once the dependency has
+/// been added other code-loading methods (LoadLibrary, CoCreateInstance, etc)
+/// can find the binaries in the resolved package.
+///
+/// Package resolution is specific to a user and can return different values
+/// for different users on a system.
+///
+/// Each successful MddAddPackageDependency() adds the resolve packaged to the
```

Comment:
resolved package, maybe you can do a global search and replace for this.

Reply:
Lolz just did after the 1st comment :-) Thanks